code/game/objects/items/weapons/storage/briefcase.dm

```diff
@@ -12,6 +12,52 @@
 	use_sound = 'sound/items/storage/briefcase.ogg'
 	drop_sound = 'sound/items/drop/backpack.ogg'
 
+	var/open = 0
+
+	examine(mob/user)

```

Comment:
No. Jesus Christ no.

Use full pathing for proc subtypes you absolute lunatic.